### DISCUSSION TITLE
GitHub Codespaces compute usage reset

### DISCUSSION THREAD

**Discussion by sakura-rains** [community/community#59798] [Codespaces]
### Select Topic Area

Product Feedback

### Body

On June 4th of last month, my mailbox received a notification that the computing resources of GitHub Codespaces were exhausted. The notification is as follows:

![image](https://github.com/community/community/assets/62538520/695d8dea-6bc1-48a7-8330-16c74ce6d5f0)

According to the information in the email, on July 1st (that is, today), my GitHub Codespaces resources will be reset, but I still cannot enter GitHub Codespaces today. What is the problem?

**Comment by samruddhikhandale** [comment]
Hi 👋 

@Sakura-rains wondering, if you are still facing the issue?
